frontend/app/views/fragments/javaScriptFirstSteps.scala.html

```diff
@@ -13,7 +13,7 @@
 <!--[if IE 9]>
 <script src="@Asset.at("javascripts/lib/polyfills.min.js")"></script>
 <![endif]-->
-<script src="https://polyfill.guim.co.uk/v2/polyfill.min.js?features=Promise,fetch"></script>
+<script src="https://polyfill.io/v3/polyfill.min.js?features=default,Array.prototype.find,fetch,Array.prototype.includes,Number.isInteger,Object.values,Object.entries"></script>
```

Comment:
dotcom uses https://assets.guim.co.uk/polyfill.io, don't know if you want to use that too?
```suggestion
<script src="https://assets.guim.co.uk/polyfill.io/v3/polyfill.min.js?features=default,Array.prototype.find,fetch,Array.prototype.includes,Number.isInteger,Object.values,Object.entries"></script>
```

Reply:
Yeah I'm happy to do thus 👍  it's probably worth us changing this on support-frontend too so we're in sync with DCR/Frontend.